### ISSUE TITLE
Design improved share dialog

### ISSUE THREAD

**Issue by ryanjduffy** [replayio/design#29]
**Updated ticket description from Jon (May 17, 2022)**
Our current share dialog has some issues that I'll describe in more detail later. [Here's a Loom](https://www.loom.com/share/46086b782f2c452b9069ca738caba192) getting people up to speed on the history and what the design proposal is moving forward.

![image](https://user-images.githubusercontent.com/9154902/169205255-31912b6c-4585-40f7-9aad-d5940ca7a73e.png)

**Original ticket from Ryan:**
There be dragons in this dialog! Changing the value in "Privacy Settings" moves the recording to a new team or makes it public/private.

**See also**
Feedback from Ollie [in this other ticket](https://github.com/RecordReplay/design/issues/43)

**Comment by jaril** [comment]
This and RecordReplay/devtools#4878 are related.

**Comment by jasonLaster** [comment]
The current experience is definitely confusing.

I can see us making some simple copy improvements in-line with what Loom has or de-coupling the teams/privacy fields.

### Simple option

* Anyone with the link can view
* Members of Replay can view
* Only people added can view

<img width="474" alt="Screen Shot 2022-01-08 at 11 13 58 AM" src="https://user-images.githubusercontent.com/254562/148656787-1d3924ab-5a80-4d44-a224-54aec7ed307e.png">

### Decoupled option

We could re-use the upload screen ui to have a similar summary and edit mode.


---

Some other considerations

* The info panel summary should be consistent with the share summary
* Clicking the info panel should jump to the share modal
* The command palette should have an option that jumps to the share modal

**Comment by jonbell-lot23** [comment]
Note while tagging tickets:

I'd consider this high importance but also more controversial than a copy change. Therefore it's harder to push out quickly, which makes other items lower hanging fruit.

**Comment by jasonLaster** [comment]
This issue https://github.com/RecordReplay/devtools/issues/3827 could potentially unblock this ui.

**Comment by jonbell-lot23** [comment]
I want to do this, but I am comfortable punting for now. We'll address later.

**Comment by jaril** [comment]
@jonbell-lot23 remind me — are we thinking of pulling this up sooner than later?

**Comment by jonbell-lot23** [comment]
A lot of things are piling up they are related to it (such as team inbox) but we haven't yet prioritised it

**Comment by jonbell-lot23** [comment]
[Here's a Loom](https://www.loom.com/share/46086b782f2c452b9069ca738caba192) showing a proposed design
[Here's us chatting in Discord](https://discord.com/channels/779097926135054346/975982613632262196)
[And here's a new ticket for the code side](https://github.com/RecordReplay/devtools/issues/6841)

**Comment by jonbell-lot23** [comment]
Marking this as complete for iteration tracking reasons
